src/liballoc/string.rs

```diff
@@ -2225,6 +2225,14 @@ impl From<&str> for String {
     }
 }
 
+#[stable(feature = "???", since = "1.43.0")]
```

Comment:
Just chose a somewhat reasonable name, for example `from_mut_str_for_string`. I also believe you have to update the Rust version to `1.44` by now.